npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/pnpm_lockfile_updater.rb

```diff
@@ -126,10 +132,23 @@ def handle_pnpm_lock_updater_error(error, pnpm_lock)
             raise_package_access_error(error_message, dependency_url, pnpm_lock)
           end
 
+          # TO-DO : subclassifcation of ERR_PNPM_TARBALL_INTEGRITY errors
+          if error_message.match?(ERR_PNPM_TARBALL_INTEGRITY)
+            dependency_names = dependencies.map(&:name).join(", ")
+
+            msg = "Error while resolving \"#{dependency_names}\"."
```

Comment:
can we include more info about the error being an unresolvable tarball or the tarball integrity? That would help.

Reply:
@abdulapopoola , i need to classify the ERR_PNPM_TARBALL_INTEGRITY errors (spent some time today but could not come with good approach _to-do_ line 135). I have added generic code for now, will work on it later.